projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-update-media-v1-1-endpoint.php

```diff
@@ -125,10 +131,45 @@ function callback( $path = '', $blog_id = 0, $media_id = 0 ) {
 			}
 		}
 
+		$result = $this->handle_video_meta( $media_id, $input, $item );
+
+		if ( is_wp_error( $result ) ) {
+			return $result;
+		}
```

Comment:
I was a bit confused here. I see that we do a check for `is_videopress_media` in `handle_video_meta`, but that was a bit confusing at first. Maybe we could add a comment here to clarify?

Reply:
Yup, sure. :) I wanted to have minimal additions in that method to abstract the behaviour, to reduce cognitive load for folks looking at the endpoint for other reasons. The actual checks is in that method `handle_video_meta`.

Reply:
Done :)